tests/postgres_tests/test_schemas.py

```diff
@@ -0,0 +1,23 @@
+from django.test import modify_settings
+
+from . import PostgreSQLTestCase
+from .models import SchemaModel
+
+
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class SchemaTest(PostgreSQLTestCase):
+
+    Model = SchemaModel
+
+    def setUp(self):
+        self.Model.objects.bulk_create([
+            self.Model(int1=1),
+            self.Model(int1=2),
+            self.Model(int1=3),
+        ])
+
+    def test_values(self):
+        index = 1
+        for object in self.Model.objects.all():
+            self.assertEqual(object.int1, index)
+            index += 1
```

Comment:
The way to test is to set a default schema for all tables in all test cases, and then check that tests pass. There will be some tests that won't pass due to checks for exact table name, those are of course OK to skip.

Unfortunately there are multiple edge cases where schema qualified tables need to be dealt with specially inside the ORM and backends. If I remember correctly, at least some queries need special handling, and then all of the database introspection framework need to be aware of the special cases. So, either all of the edge cases need to be fixed, or at least we need to know which edge cases do not work.